commit 699a57e13dbac03af16cfd49d838cd233d189dbd
Author: Gwenole Beauchesne <gbeauchesne@splitted-desktop.com>
Date:   Tue Apr 13 04:54:11 2010 +0200

    Add OpenGL extensions (v3) and generic implementation with TFP and FBO.

diff --git a/va/glx/va_glx_impl.h b/va/glx/va_glx_impl.h
index fbdcac2..ca1095e 100644
--- a/va/glx/va_glx_impl.h
+++ b/va/glx/va_glx_impl.h
@@ -31,7 +31,7 @@
  * @param[in]  ctx        the VA driver context
  * @return VA_STATUS_SUCCESS if successful
  */
-VAStatus va_glx_init_context(VADriverContextP ctx);
-    /* ATTRIBUTE_HIDDEN; */
+VAStatus va_glx_init_context(VADriverContextP ctx)
+    ATTRIBUTE_HIDDEN;
 
 #endif /* VA_GLX_IMPL_H */
